lib/taskana-core/src/main/java/pro/taskana/common/internal/JobMapper.java

```diff
@@ -25,6 +25,9 @@ public interface JobMapper {
           + "<when test=\"_databaseId == 'db2'\">"
           + "SCHEDULED_JOB_SEQ.NEXTVAL"
           + "</when>"
+          + "<when test=\"_databaseId == 'oracle'\">"
+          + "SCHEDULED_JOB_SEQ.NEXTVAL"
+          + "</when>"
```

Comment:
idea: reuse the previous test statement? Is this better? What do you think?

Reply:
maybe this can work?
![Screenshot from 2023-01-31 16-21-28](https://user-images.githubusercontent.com/15628173/215801476-d43b1889-fcec-46f0-827b-7a80cb4f7ebb.png)

Reply:
thank you